webapp/dev/blocks/api/api.js

```diff
@@ -1,6 +1,6 @@
 var API = (function() {
 	var that = this,
-		basePath = 'https://safesecret.info/api/v1/';
+		basePath = 'https://' + location.hostname + '/api/v1/';
```

Comment:
I think you should use `location.origin` here.

```
basePath = location.origin + '/api/v1/';
```

Reply:
Done